docs/evaluate/development-features/versioning.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: versioning
+title: Versioning - Temporal Development Feature
+description: Learn how to implement versioning in your Temporal Workflows to manage changes in application code without causing non-deterministic behavior. Explore the benefits of versioning and how it ensures the continuity of long-running Workflows.
+sidebar_label: Versioning
+tags:
+  - versioning
+  - Workflow-versioning
+  - temporal-Workflows
+  - version-management
+keywords:
+  - temporal versioning
+  - manage Workflow changes
+  - version Workflow executions
+  - temporal versioning best practices
+  - Workflow continuity temporal
+  - version management
+  - temporal Workflow versioning
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Versioning in Temporal is a feature that allows safely evolve your Temporal application code in production.
+Developers can make changes to their application code without causing non-deterministic behavior in current long-running Workflows. 
+
+Workflow Versioning allows you to safely deploy incompatible changes to your Workflow code.
+This is especially important when you have Workflow Executions that were started with the original code and are still running when you deploy the updated code.
+
+Depending on your use case, the longevity of your Workflow Executions, and your organizational overhead, you can implement Versioning by changing Workflow Types or by using the Patching API. 
```

Comment:
I think since this is the landing page, we get away from the technicalities of how to implement versioning and just talk about what it is.